src/lbgrid.cc

```diff
@@ -7,6 +7,14 @@ lbgrid::lbgrid(int nx, int ny) : nx_{nx}, ny_{ny} {
   }
 }
 
+int lbgrid::nx_value() {
```

Comment:
Typically these unmodifying functions should be `const` everything so it will be `int lbgrid::nx() const {`